src/server/local/index.test.js

```diff
@@ -0,0 +1,281 @@
+const express = require('express');
+const request = require('supertest');
+const path = require('path');
+const local = require('./index').default;
+
+const sendFileSpy = jest.spyOn(express.response, 'sendFile');
+
+const makeRequest = async requestPath =>
+  request(local(express())).get(requestPath);
+
+const DATA_DIRECTORY = path.join(process.cwd(), 'data');
+
+const assertDataFile = ({ dataPath, expectedDataFile }) => {
+  it(`should be served for ${dataPath}`, async () => {
+    const response = await makeRequest(dataPath);
+    expect(response.status).toBe(200);
+    expect(
+      sendFileSpy.mock.calls[sendFileSpy.mock.calls.length - 1][0],
+    ).toEqual(path.join(DATA_DIRECTORY, expectedDataFile));
+    sendFileSpy.mockClear();
+  });
+};
+
+describe('Local Server', () => {
+  describe('Article data file', () => {
+    const articleDataPaths = [
+      '/news/articles/c0000000001o.json',
+      '/zhongwen/articles/c3xd4x9prgyo/simp.json',
+    ];
+
+    articleDataPaths.forEach(articleDataPath => {
+      assertDataFile({
+        dataPath: articleDataPath,
+        expectedDataFile: articleDataPath,
+      });
+    });
+  });
+
+  describe('Front Page data file', () => {
+    const frontPageDataPaths = [
```

Comment:
Most of these tests are very similar. Could we use the `it.each` logic to reduce the number of repeated tests?

Reply:
Similar to test here `/src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/buildIncludeUrl/index.test.js`

Reply:
Done! Thanks for the signpost, it was very helpful 😄